### ISSUE TITLE
Clean up console messages

### ISSUE THREAD

**Issue by tonydisera** [iobio/genepanel.iobio#128]

**Comment by tonydisera** [comment]
When I am trying to debug all of the apps in clin.iobio, it is hard with all of the console message coming from genepanel.iobio.  Can you please remove all of messages that are no longer needed for debugging?

For example, these messages are showing in the console:

'searchUrl https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=medgen&usehistory=y&retmode=json&term=(((Pyruvate dehydrogenase complex deficiency[title]) AND "in gtr"[Filter])) AND (("conditions"[Filter] OR "diseases"[Filter]))
Model.js:82 data is :  Object
Model.js:93 summaryUrl is :  https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=gtr&query_key=1&WebEnv=NCID_1_28615614_130.14.18.34_9001_1526682204_2126813252_0MetA0_S_MegaStore&usehistory=y
Model.js:105 Results in model.js for search terms are  Object'

**Comment by adityaekawade** [comment]
Yes, I removed many of them today. I will clean up the remaining ones.
